### PR TITLE
Adding TERMUX_PKG_GROUPS value for pacman packages.

### PR DESCRIPTION
**Pull request by Maxython**
Thanks to this value, can set the group that the package will belong to.  
#### Example:
```
TERMUX_PKG_GROUPS="base-devel"
```
or
```
TERMUX_PKG_GROUPS="base-devel, base"
```
If anything is there in more detail: https://wiki.archlinux.org/title/Pacman#Installing_package_groups

### CODE REVIEW

File: scripts/build/termux_create_pacman_subpackages.sh [hunk 1]
```diff
@@ -107,6 +107,10 @@ termux_create_pacman_subpackages() {
 			if [ -n "$TERMUX_SUBPKG_CONFFILES" ]; then
 				tr ',' '\n' <<< "$TERMUX_SUBPKG_CONFFILES" | awk '{ printf "backup = '"${TERMUX_PREFIX:1}"'/%s\n", $1 }'
 			fi
+
+			if [ -n "$TERMUX_PKG_GROUPS" ]; then
+				tr ',' '\n' <<< "${TERMUX_PKG_GROUPS/#, /}" | awk '{ printf "group = %s\n", $1 }'
```
Comment (xeffyr) [inline scripts/build/termux_create_pacman_subpackages.sh:112]: Why you are adding this only for subpackages?

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: scripts/build/termux_create_pacman_subpackages.sh (hunk 1 above)
Comment (Maxython) [reply scripts/build/termux_create_pacman_subpackages.sh:112]: I only added subpackages so that the groups only have the basics of the packages.

Comment (xeffyr) [reply scripts/build/termux_create_pacman_subpackages.sh:112]: This means you will not be able to specify `automake`, `autoconf` and other packages as `base-devel` or `bash`, `coreutils` as `base` - just for example. Setting groups only for splits will make them barely useless.

Comment (Maxython) [reply scripts/build/termux_create_pacman_subpackages.sh:112]: Ok

**Review by xeffyr — APPROVED** [review]

**Pull request merged**